platform-sdk/swirlds-platform-core/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ plugins {
 mainModuleInfo {
     annotationProcessor("com.swirlds.config.processor")
     runtimeOnly("com.swirlds.config.impl")
+    runtimeOnly("com.swirlds.logging.log4j.appender")
```

Comment:
That line would add our appender everywhere. I assume that is not what we want for now, right?